### ISSUE TITLE
fixed column is not aligned

### ISSUE THREAD

**Issue by pntcwz** [markmalek/Fixed-Header-Table#119]
Hi, thanks for this great plugin.
My problem is fixed column is not aligned.
like this image in demo file
![2014-04-16_200217](https://cloud.githubusercontent.com/assets/6553964/2719320/fd11c61c-c55e-11e3-854c-dd2d9e6f8963.png)

i opened by ie chrome firefox, but only in firefox run well.

Thank you in advance.

**Comment by jxramos** [comment]
I don't think this issue has been solved yet judging by the demo page demonstrating this bug.

There should be a workaround looking at how meetselva does it on line 171-187 for b512f9a  on https://github.com/meetselva/fixed-table-rows-cols/blob/master/js/fixed_table_rc.js 

The meetselva solution has an analogous misalignment problem but in the column widths of the fixed row, not the row heights of the fixed column which Mark Malek's tool seems to suffer from.

![fixedcolumnheights](https://cloud.githubusercontent.com/assets/13857038/26017921/e7f7dc94-3720-11e7-8f6f-8fdf1f0f2ece.png)
